MapboxDirections/MBDirectionsOptions.swift

```diff
@@ -444,6 +444,11 @@ open class DirectionsOptions: NSObject, NSSecureCoding, NSCopying {
             params.append(URLQueryItem(name: "annotations", value: attributesStrings))
         }
         
+        if !waypoints.compactMap({ $0.name }).isEmpty {
+            let names = waypoints.map { $0.name ?? "" }.joined(separator: ";")
```

Comment:
Would it be possible to do this map and compactMap once?

Reply:
No, because an intermediate waypoint can have no name, but it would still need an empty entry in the semicolon-delimited list.